.eslintrc.json

```diff
@@ -34,6 +34,10 @@
     "linebreak-style": ["error", "unix"],
     "semi": ["error", "never"],
     "@typescript-eslint/no-non-null-assertion": "off",
-    "@typescript-eslint/no-var-requires": "off"
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "no-extra-boolean-cast": "off",
+    "no-undef": "off"
```

Comment:
I think it would be good to have this last three rules on, if it's not too much of a pain in the 🍑 ... WDYT?

Reply:
yea, looks like this is not fixing prettier errors... 😆